App.java

```diff
@@ -0,0 +1,22 @@
+public class App {
+    public static String getCallerClassAndMethodName() {
+        StackTraceElement[] stackTraceElements = new Exception().getStackTrace();
+        try {
+            return stackTraceElements[2].getClassName()
+                    + " > "
```

Comment:
optional: for stepik we need use:
```suggestion
                    + "#"
```